dotnet/samples/KernelSyntaxExamples/Example15_TextMemoryPlugin.cs

```diff
@@ -156,7 +156,7 @@ private static async Task RunWithStoreAsync(IMemoryStore memoryStore, Cancellati
             .Build();
 
         // Create an embedding generator to use for semantic memory.
-        var embeddingGenerator = new OpenAITextEmbeddingGeneration(TestConfiguration.OpenAI.EmbeddingModelId, TestConfiguration.OpenAI.ApiKey);
+        var embeddingGenerator = new OpenAITextEmbeddingGenerationService(TestConfiguration.OpenAI.EmbeddingModelId, TestConfiguration.OpenAI.ApiKey);
```

Comment:
We are adding Service just for the interfaces, I think the implementations don't necessarily need the suffix.

Reply:
We have added them elsewhere e.g. https://github.com/microsoft/semantic-kernel/blob/main/dotnet/src/Connectors/Connectors.OpenAI/ChatCompletion/AzureOpenAIChatCompletionService.cs